src/components/Top.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { Button } from 'reactstrap';
+import { Link, withRouter, Redirect} from 'react-router-dom';
+
+class Top extends Component {
+  onClickNewGame(){
+    this.props.createGame();
+    this.props.history.push('/');
```

Comment:
I wonder what this code mean. 
I think when createButton clicked, the flow is like following. Is it correct?

1. call createGame and set props.roomId by redux.(calling dispatch)
2. In render method, when props.roomId has value, Redirect to `/map`